### PR TITLE
Replace pkg/errors wrapping with fmt.Errorf

### PR DESCRIPTION
**Pull request by u5surf**
## Which problem is this PR solving?
- #2061 

## Short description of the changes
- I replaced error.wrapping using `fmt.Errorf`

### CODE REVIEW

File: cmd/agent/app/builder_test.go [hunk 1]
```diff
@@ -182,7 +181,7 @@ func (fakeCollectorProxy) EmitBatch(batch *jaeger.Batch) (err error) {
 }
 
 func (f fakeCollectorProxy) GetSamplingStrategy(serviceName string) (*sampling.SamplingStrategyResponse, error) {
-	return nil, errors.New("no peers available")
+	return nil, fmt.Errorf("no peers available")
```
Comment (yurishkuro) [inline cmd/agent/app/builder_test.go]: there's no need to change this

**Comment by u5surf** [comment]
> There are many unnecessary changes here. The intention was to remove the use of `"github.com/pkg/errors"`, but there's nothing wrong with using the standard `errors.New()`.
> 
> To ensure that `"github.com/pkg/errors"` is gone, we should remove it from Gopkg.toml

copied

**Comment by u5surf** [comment]
@yurishkuro 
I replaced `github.com/pkg/errors` with native libraries.
then I confirmed following `make lint` error.  It is easy to resolve this error, but some error messages will be changed the non capitalized starting. May I fix all of them?
```
$ make lint
time staticcheck ./... \
        | grep -v \
                -e model/model.pb.go \
                -e thrift-gen/ \
        >> .lint.log || true

real    0m14.487s
user    0m18.184s
sys     0m2.832s
Detected staticcheck failures:
cmd/agent/app/reporter/grpc/collector_proxy.go:24:2: package "github.com/jaegertracing/jaeger/cmd/agent/app/reporter" is being imported more than once (ST1019)
        cmd/agent/app/reporter/grpc/collector_proxy.go:25:2: other import of "github.com/jaegertracing/jaeger/cmd/agent/app/reporter"
pkg/es/config/config.go:90:15: error strings should not be capitalized (ST1005)
pkg/kafka/auth/config.go:66:10: error strings should not end with punctuation or a newline (ST1005)
plugin/storage/badger/dependencystore/storage_test.go:83:21: unnecessary use of fmt.Sprintf (S1039)
plugin/storage/badger/spanstore/read_write_test.go:430:19: unnecessary use of fmt.Sprintf (S1039)
plugin/storage/badger/spanstore/read_write_test.go:432:18: unnecessary use of fmt.Sprintf (S1039)
plugin/storage/cassandra/spanstore/reader.go:75:25: error strings should not be capitalized (ST1005)
plugin/storage/cassandra/spanstore/reader.go:78:34: error strings should not be capitalized (ST1005)
plugin/storage/cassandra/spanstore/reader.go:81:33: error strings should not be capitalized (ST1005)
plugin/storage/cassandra/spanstore/reader.go:84:30: error strings should not be capitalized (ST1005)
plugin/storage/cassandra/spanstore/reader.go:87:39: error strings should not be capitalized (ST1005)
plugin/storage/cassandra/spanstore/reader.go:90:29: error strings should not be capitalized (ST1005)
plugin/storage/es/dependencystore/storage.go:103:16: error strings should not be capitalized (ST1005)
plugin/storage/es/spanstore/reader.go:67:25: error strings should not be capitalized (ST1005)
plugin/storage/es/spanstore/reader.go:70:34: error strings should not be capitalized (ST1005)
plugin/storage/es/spanstore/reader.go:73:33: error strings should not be capitalized (ST1005)
plugin/storage/es/spanstore/reader.go:76:30: error strings should not be capitalized (ST1005)
plugin/storage/es/spanstore/reader.go:79:29: error strings should not be capitalized (ST1005)
plugin/storage/es/spanstore/reader.go:82:38: error strings should not be capitalized (ST1005)
plugin/storage/es/spanstore/reader.go:277:16: error strings should not be capitalized (ST1005)
plugin/storage/es/spanstore/service_operation.go:97:15: error strings should not be capitalized (ST1005)
plugin/storage/es/spanstore/service_operation.go:128:15: error strings should not be capitalized (ST1005)
make: *** [lint-staticcheck] error 1
```

**Comment by yurishkuro** [comment]
it's ok to change capitalization.  but first please see my comment https://github.com/jaegertracing/jaeger/pull/2070/files#r378924852 - you're making too many unnecessary changes by replacing "errors" with "fmt", which is not in scope of this ticket.

File: Gopkg.toml [hunk 2]
```diff
@@ -83,10 +83,6 @@ required = [
   name = "github.com/opentracing/opentracing-go"
   version = "^1.0.2"
 
-[[constraint]]
-  name = "github.com/pkg/errors"
-  version = "^0.8.0"
-
```
Comment (yurishkuro) [inline Gopkg.toml]: make sure to run `dep ensure --update` to reflect this change in the lock file.

File: plugin/storage/es/spanstore/service_operation.go [hunk 3]
```diff
@@ -87,14 +87,14 @@ func (s *ServiceOperationStorage) getServices(context context.Context, indices [
 
 	searchResult, err := searchService.Do(context)
 	if err != nil {
-		return nil, errors.Wrap(err, "Search service failed")
+		return nil, fmt.Errorf("search service failed: %w", err)
```
Comment (yurishkuro) [inline plugin/storage/es/spanstore/service_operation.go]: nit: `search for services failed`

File: plugin/storage/es/spanstore/service_operation.go [hunk 4]
```diff
@@ -118,14 +118,14 @@ func (s *ServiceOperationStorage) getOperations(context context.Context, indices
...
```
Comment (yurishkuro) [inline plugin/storage/es/spanstore/service_operation.go]: nit: `search for operations failed`

**Comment by yurishkuro** [comment]
The linter is breaking due to upgrade of grpc. I am adjusting the deps separately in #2072.

**Comment by yurishkuro** [comment]
NB: you may want to undo the `Gopkg.*` changes, because pkg/errors is still a transitive dependency, and not having to touch deps will make this PR less complicated.

**Comment by u5surf** [comment]
@yurishkuro Certainly, Gopkg.* reverted.

**Comment by yurishkuro** [comment]
Thank you!

**Pull request merged**